README.rst

```diff
@@ -35,15 +32,28 @@ Getting started
 
 2. Add ``wagtailtrans`` to your INSTALLED_APPS
 
-3. Perform a migration
+3. Add `WAGTAILTRANS_TEMPLATE_DIR` (from wagtailtrans import WAGTAILTRANS_TEMPLATE_DIR) to your template dirs setting
+
+.. code-block:: python
+    from wagtailtrans import WAGTAILTRANS_TEMPLATE_DIR
+
+    TEMPLATES = [{
+        # ....
+        'DIRS': [
+            WAGTAILTRANS_TEMPLATE_DIRS,
```

Comment:
Adding the `WAGTAILTRANS_TEMPLATE_DIRS` causes the delete page functionality to error out on *Non translatable pages* normal page.

<img width="901" alt="screen shot 2016-11-09 at 6 57 54 pm" src="https://cloud.githubusercontent.com/assets/1276650/20140019/dc380030-a6ae-11e6-95da-4af8d6e82abc.png">